test/unit/spans/map-to-streaming-type.test.js

```diff
@@ -46,7 +46,7 @@ test('should correctly convert bools when false', async () => {
 })
 
 test('should correctly convert integers', async () => {
-  const intValue = 9999999999999999
+  const intValue = 999999999999999
```

Comment:
This one generated an error about the value overflowing. I don't think we were trying to test against an overflowed value, so I just shortened it to fit.